bambi/models.py

```diff
@@ -812,20 +812,20 @@ def predict(
         idata : InferenceData
             The ``InferenceData`` instance returned by ``.fit()``.
         kind : str
-            Indicates the type of prediction required. Can be ``"mean"`` or ``"pps"``. The
-            first returns draws from the posterior distribution of the mean, while the latter
-            returns the draws from the posterior predictive distribution (i.e. the posterior
-            probability distribution for a new observation) in addition to the mean posterior
-            distribution. Defaults to ``"mean"``.
+            Indicates the type of prediction required. Can be ``"response_params"`` or
+            ``"response"``. The first returns draws from the posterior distribution of
+            the mean, while the latter returns the draws from the posterior predictive
```

Comment:
Can it be "from the posterior distribution of the parameters of the likelihood"? As those parameters could be something different from the mean